package.json

```
@@ -29,7 +29,7 @@
   "dependencies": {
     "@elastic/elasticsearch": "8.12.0",
     "@google-cloud/storage": "7.7.0",
-    "axios": "1.8.2",
+    "axios": "^1.12.2",
```

```suggestion
    "axios": "1.12.2",
```

We prefer explicitly specified dependency versions for applications